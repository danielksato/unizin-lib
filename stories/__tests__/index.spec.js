import initstoryshots from '@storybook/addon-storyshots';

initstoryshots();
