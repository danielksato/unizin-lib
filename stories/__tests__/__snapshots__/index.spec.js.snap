// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Add Remove Button Button 1`] = `
<button
  className="is-primary sc-EHOje idiwCv"
  onClick={[Function]}
>
  <span
    className="cross"
  />
  <span
    className="sc-ifAKCX izpMtP"
  >
    Add
     
    Bork bork bork
  </span>
</button>
`;

exports[`Storyshots AvatarOption Elvis 1`] = `
<div
  className="sc-bxivhb ivbKJD"
>
  <div
    className="sc-bwzfXH iQOhsx"
  >
    <img
      alt="Gravatar image for elvis@example.com"
      className="avatar sc-htpNat iCxrQH"
      src="https://www.gravatar.com/avatar/73bd673a3291d9a3ea9857b773f75440?d=blank"
    />
    <div
      aria-hidden={true}
      className="sc-bdVaJa fnrRkl"
    >
      EP
    </div>
  </div>
  <span>
    Elvis Presley
  </span>
</div>
`;

exports[`Storyshots Content Pricing Free content 1`] = `
<div
  className="sc-bZQynM Agekg"
>
  <div>
    <div
      className="sc-gzVnrw hsEcyB"
    >
      <span
        className="sc-ifAKCX izpMtP"
      >
        Your price
      </span>
       
      <span
        className="sc-dnqmqq cdIqsh"
      >
        Free
      </span>
    </div>
    <div
      className="sc-gZMcBi hBmVNX"
    >
      <span
        className="sc-iwsKbI jMxFSW"
      >
        $
        174.95
      </span>
       (
      100
      % savings)
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Pricing Free content, missing list price 1`] = `
<div
  className="sc-bZQynM Agekg"
>
  <div>
    <div
      className="sc-gzVnrw hsEcyB"
    >
      <span
        className="sc-ifAKCX izpMtP"
      >
        Your price
      </span>
       
      <span
        className="sc-dnqmqq cdIqsh"
      >
        Free
      </span>
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Pricing Has all information 1`] = `
<div
  className="sc-bZQynM Agekg"
>
  <div>
    <div
      className="sc-gzVnrw hsEcyB"
    >
      <span
        className="sc-ifAKCX izpMtP"
      >
        Your price
      </span>
       
      <span
        className="sc-dnqmqq cdIqsh"
      >
        $74.99
      </span>
    </div>
    <div
      className="sc-gZMcBi hBmVNX"
    >
      <span
        className="sc-iwsKbI jMxFSW"
      >
        $
        174.95
      </span>
       (
      57
      % savings)
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Pricing Missing both prices (why do we allow this?) 1`] = `
<div>
  <div
    aria-hidden={true}
  >
    N/A
  </div>
  <div
    className="aural"
  >
    Pricing not available
  </div>
</div>
`;

exports[`Storyshots Content Pricing Missing list price 1`] = `
<div
  className="sc-bZQynM Agekg"
>
  <div>
    <div
      className="sc-gzVnrw hsEcyB"
    >
      <span
        className="sc-ifAKCX izpMtP"
      >
        Your price
      </span>
       
      <span
        className="sc-dnqmqq cdIqsh"
      >
        $74.99
      </span>
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Pricing Missing offer price 1`] = `
<div>
  <div
    aria-hidden={true}
  >
    N/A
  </div>
  <div
    className="aural"
  >
    Pricing not available
  </div>
</div>
`;

exports[`Storyshots Loading Loading 1`] = `
<div
  className="sc-gqjmRU fYdCyB"
>
  Loading
  ...
</div>
`;

exports[`Storyshots ModalPortal (Confirm modal) Base props 1`] = `
<div
  className="sc-jTzLTM ccDZgE"
  tabIndex="0"
>
  <main
    className="sc-fjdhpX kxXcJy"
  >
    <button
      aria-label="Close"
      className="sc-jzJRlG hdLZni sc-VigVT iyDkum"
      onClick={[Function]}
    />
    <div
      className="sc-kAzzGY gDahZl"
    >
      <h1>
        Would you like a cookie?
      </h1>
      <p>
        They are very tasty
      </p>
    </div>
    <div
      className="sc-cSHVUG lcKYNk"
    >
      <button
        onClick={[Function]}
      >
        No
      </button>
      <button
        onClick={[Function]}
      >
        Yes
      </button>
    </div>
  </main>
</div>
`;

exports[`Storyshots ModalPortal (Confirm modal) With custom labeling 1`] = `
<div
  className="sc-jTzLTM ccDZgE"
  tabIndex="0"
>
  <main
    className="sc-fjdhpX kxXcJy"
  >
    <button
      aria-label="Close"
      className="sc-jzJRlG hdLZni sc-VigVT iyDkum"
      onClick={[Function]}
    />
    <div
      className="sc-kAzzGY gDahZl"
    >
      <h1>
        Would you like a cookie?
      </h1>
      <p>
        They are very tasty
      </p>
    </div>
    <div
      className="sc-cSHVUG lcKYNk"
    >
      <button
        onClick={[Function]}
      >
        Bork
      </button>
      <button
        onClick={[Function]}
      >
        Bork bork bork
      </button>
    </div>
  </main>
</div>
`;

exports[`Storyshots ModalPortal (Confirm modal) Without cancel 1`] = `
<div
  className="sc-jTzLTM ccDZgE"
  tabIndex="0"
>
  <main
    className="sc-fjdhpX kxXcJy"
  >
    <button
      aria-label="Close"
      className="sc-jzJRlG hdLZni sc-VigVT iyDkum"
      onClick={[Function]}
    />
    <div
      className="sc-kAzzGY gDahZl"
    >
      <h1>
        Would you like a cookie?
      </h1>
      <p>
        They are very tasty
      </p>
    </div>
    <div
      className="sc-cSHVUG lcKYNk"
    >
      <button
        onClick={[Function]}
      >
        Yes
      </button>
    </div>
  </main>
</div>
`;

exports[`Storyshots Notifications All three notifications 1`] = `
<div
  className="sc-kGXeez uuwYj"
>
  <div
    className="sc-kgoBCf kCAiLg"
    role="status"
  >
    <div
      style={
        Object {
          "color": "#D53A60",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times-circle fa-w-16 "
        data-icon="times-circle"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <span>
      Cancel Notification
    </span>
    <button
      className="sc-chPdSV isHmvr sc-VigVT iyDkum"
      label="Dismiss"
      onClick={[Function]}
    />
  </div>
  <div
    className="sc-kgoBCf kCAiLg"
    role="status"
  >
    <div
      style={
        Object {
          "color": "#45DC85",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-check-circle fa-w-16 "
        data-icon="check-circle"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <span>
      Check Notification
    </span>
    <button
      className="sc-chPdSV isHmvr sc-VigVT iyDkum"
      label="Dismiss"
      onClick={[Function]}
    />
  </div>
  <div
    className="sc-kgoBCf kCAiLg"
    role="status"
  >
    <div
      style={
        Object {
          "color": "#FFD226",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-exclamation-triangle fa-w-18 "
        data-icon="exclamation-triangle"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 576 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <span>
      Exclamation Notification
    </span>
    <button
      className="sc-chPdSV isHmvr sc-VigVT iyDkum"
      label="Dismiss"
      onClick={[Function]}
    />
  </div>
</div>
`;

exports[`Storyshots Notifications Call to action 1`] = `
<div
  className="sc-kGXeez uuwYj"
>
  <div
    className="sc-kgoBCf kCAiLg"
    role="status"
  >
    <div
      style={
        Object {
          "color": "#45DC85",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-check-circle fa-w-16 "
        data-icon="check-circle"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <span>
      Check Notification
    </span>
    <button
      onClick={[Function]}
    >
      Do a thing
    </button>
    <button
      className="sc-chPdSV isHmvr sc-VigVT iyDkum"
      label="Dismiss"
      onClick={[Function]}
    />
  </div>
</div>
`;

exports[`Storyshots Notifications No dismiss 1`] = `
<div
  className="sc-kGXeez uuwYj"
>
  <div
    className="sc-kgoBCf kCAiLg"
    role="alert"
  >
    <div
      style={
        Object {
          "color": "#45DC85",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-check-circle fa-w-16 "
        data-icon="check-circle"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <span>
      Check Notification
    </span>
  </div>
</div>
`;

exports[`Storyshots Pagination On Page 1 1`] = `
<div
  className="sc-kpOJdX lgchly"
>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={true}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-left fa-w-8 "
        data-icon="angle-left"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Previous page
      </span>
    </button>
  </div>
  <div>
    Page 
    1
     of 
    2
  </div>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={false}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-right fa-w-8 "
        data-icon="angle-right"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Next page
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots Pagination doensn't have next, on Page 1 1`] = `null`;

exports[`Storyshots Pagination doensn't have next, on Page 2 1`] = `
<div
  className="sc-kpOJdX lgchly"
>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={false}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-left fa-w-8 "
        data-icon="angle-left"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Previous page
      </span>
    </button>
  </div>
  <div>
    Page 
    2
     of 
    2
  </div>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={true}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-right fa-w-8 "
        data-icon="angle-right"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Next page
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots Pagination hasNext, on Page 2 1`] = `
<div
  className="sc-kpOJdX lgchly"
>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={false}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-left fa-w-8 "
        data-icon="angle-left"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Previous page
      </span>
    </button>
  </div>
  <div>
    Page 
    2
     of 
    3
  </div>
  <div>
    <button
      className="sc-dxgOiQ cflSXG"
      disabled={false}
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="svg-inline--fa fa-angle-right fa-w-8 "
        data-icon="angle-right"
        data-prefix="fas"
        role="img"
        style={Object {}}
        viewBox="0 0 256 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="sc-ifAKCX izpMtP"
      >
        Next page
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots ToggleHandle Toggleable handle 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
    }
  }
>
  <div
    className="sc-ckVGcZ hYkLcI"
    onClick={[Function]}
  />
</div>
`;

exports[`Storyshots Tooltip Tooltip with UA text 1`] = `
<div
  className="sc-kEYyzF kDpGAm"
>
  <div
    className="ra-tooltip-wrapper sc-hMqMXs dwxBtC"
    id="ra-tooltip-1"
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    role="tooltip"
  >
    <div
      aria-hidden={true}
      className="bottom ra-tooltip sc-eNQAEJ eEqcsN"
    >
      <div
        className="sc-jKJlTe cFdvXR ra-tooltip-message"
      >
        <p>
          UA text goes here.
        </p>
      </div>
    </div>
    <svg
      aria-describedby="ra-tooltip-1"
      aria-hidden="true"
      className="svg-inline--fa fa-question-circle fa-w-16 "
      data-icon="question-circle"
      data-prefix="fas"
      role="img"
      style={Object {}}
      tabIndex={0}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;

exports[`Storyshots ZingTouch Allow native zoom 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "yellow",
        "height": 400,
        "width": 400,
      }
    }
  />
</div>
`;

exports[`Storyshots ZingTouch Gesture handlers 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "yellow",
        "height": 400,
        "width": 400,
      }
    }
  />
</div>
`;

exports[`Storyshots ZingTouch Pass zingTouchRegion prop 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "blue",
        "height": 400,
        "width": 400,
      }
    }
  />
</div>
`;
